geode-core/src/distributedTest/java/org/apache/geode/internal/cache/eviction/EvictionDUnitTest.java

```diff
@@ -102,6 +104,14 @@ public void before() {
     }, server0, server1);
   }
 
+  @After
```

Comment:
I don't think we need to be using bounce to address the flakiness. We need to see where the calculation is getting wrong; if its with initial memory (before the test runs) or the memory usage/leak as the tests gets run and accordingly calculate the eviction. If its memory leak, need to address that. 
For now we could make this change to address the flakiness in the ci pipeline, but we should keep the ticket open with the comments to reproduce the issue.

Reply:
Will keep the ticket open